### PR TITLE
[Documentation] Fix typos and broken links

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
Fix and improve the documentation.

### Changes
Fixes of several typos and broken links. Additionally, the styling in some files were improved to match the rest of the documentation.

### Tests
This does not apply to the changes in this PR.

### Feature Plan
There are not any remaining steps left.

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [x] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: docs/_docs/reference/skills/bestpractices.md [hunk 1]
```diff
@@ -282,7 +285,7 @@ private async Task<List<Card>> GetMeetingCardListAsync(DialogContext dc, List<Ev
     var state = await Accessor.GetAsync(dc.Context);
 
     var eventItemList = new List<Card>();
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 2]
```diff
@@ -266,9 +269,9 @@ In the Point of Interest Skill, a route state model is passed to a Microsoft.Bot
         TravelDelaySpeak = GetFormattedTrafficDelayString(trafficTimeSpan)
     };
 
-    // Instantiate a new Card with reference to above JSON
-    var card = new Card("RouteDirectionsViewCard", routeDirectionsModel);
-
+	// Instantiate a new Card with reference to above JSON
+	var card = new Card("RouteDirectionsViewCard", routeDirectionsModel);
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 3]
```diff
@@ -299,14 +302,14 @@ private async Task<List<Card>> GetMeetingCardListAsync(DialogContext dc, List<Ev
                 }
             });
         }
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 4]
```diff
@@ -299,14 +302,14 @@ private async Task<List<Card>> GetMeetingCardListAsync(DialogContext dc, List<Ev
...
-
+    
         eventItemList.Add(new Card()
         {
             Name = "CalendarItem",
             Data = item.ToAdaptiveCardData(state.GetUserTimeZone())
         });
     }
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 5]
```diff
@@ -340,9 +343,9 @@ protected async Task<Activity> GetOverviewMeetingListResponseAsync(
             Indicator = string.Format(CalendarCommonStrings.ShowMeetingsIndicator, (firstIndex + 1).ToString(), lastIndex.ToString(), totalCount.ToString())
         }
     };
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 6]
```diff
@@ -340,9 +343,9 @@ protected async Task<Activity> GetOverviewMeetingListResponseAsync(
             Indicator = string.Format(CalendarCommonStrings.ShowMeetingsIndicator, (firstIndex + 1).ToString(), lastIndex.ToString(), totalCount.ToString())
         }
     };
-
+    
     var eventItemList = await GetMeetingCardListAsync(dc, events);
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 7]
```diff
@@ -367,16 +370,16 @@ protected PromptOptions GetPointOfInterestChoicePromptOptions(List<PointOfIntere
             {
                 var item = pointOfInterestList[i].Name;
                 var address = pointOfInterestList[i].Street;
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 8]
```diff
@@ -367,16 +370,16 @@ protected PromptOptions GetPointOfInterestChoicePromptOptions(List<PointOfIntere
...
-
+    
                 List<string> synonyms = new List<string>()
                     {
                         item,
                         address,
                         (i + 1).ToString(),
                     };
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 9]
```diff
@@ -367,16 +370,16 @@ protected PromptOptions GetPointOfInterestChoicePromptOptions(List<PointOfIntere
...
                     {
                         item,
                         address,
                         (i + 1).ToString(),
                     };
-
+    
                 var suggestedActionValue = item;
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 10]
```diff
@@ -388,15 +391,15 @@ protected PromptOptions GetPointOfInterestChoicePromptOptions(List<PointOfIntere
                         };
                     suggestedActionValue = ResponseManager.GetResponse(promptTemplate, promptReplacements).Text;
                 }
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 11]
```diff
@@ -388,15 +391,15 @@ protected PromptOptions GetPointOfInterestChoicePromptOptions(List<PointOfIntere
...
+    
                 var choice = new Choice()
                 {
                     Value = suggestedActionValue,
                     Synonyms = synonyms,
                 };
                 options.Choices.Add(choice);
             }
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 12]
```diff
@@ -433,12 +436,12 @@ protected async Task<bool> ChoiceValidator(PromptValidatorContext<FoundChoice> p
             return true;
         }
     }
-
-    return false;
+    
+	return false;
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 13]
```diff
@@ -458,13 +461,13 @@ protected async Task HandleDialogExceptions(WaterfallStepContext sc, Exception e
 
     // log exception
     TelemetryClient.TrackExceptionEx(ex, sc.Context.Activity, sc.ActiveDialog?.Id);
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 14]
```diff
@@ -458,13 +461,13 @@ protected async Task HandleDialogExceptions(WaterfallStepContext sc, Exception e
 
     // log exception
     TelemetryClient.TrackExceptionEx(ex, sc.Context.Activity, sc.ActiveDialog?.Id);
-
+    
     // send error message to bot user
     await sc.Context.SendActivityAsync(ResponseManager.GetResponse(SharedResponses.ErrorMessage));
-
+
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/bestpractices.md [hunk 15]
```diff
@@ -458,13 +461,13 @@ protected async Task HandleDialogExceptions(WaterfallStepContext sc, Exception e
...
-
+    
     // send error message to bot user
     await sc.Context.SendActivityAsync(ResponseManager.GetResponse(SharedResponses.ErrorMessage));
-
+    
     // clear state
     var state = await ConversationStateAccessor.GetAsync(sc.Context);
-    state.Clear();
+	state.Clear();
```
Comment (dfavretto) [inline docs/_docs/reference/skills/bestpractices.md]: Is this indentation correct?

File: docs/_docs/reference/skills/phone.md [hunk 16]
```diff
@@ -74,7 +73,7 @@ The Phone Skill requires the following dependencies for end to end operation, wh
 
 > Review the pricing and terms for the services and adjust to suit your scenario.
 
-To deploy your services using the default configuration, follow the steps in this common [deployment documentation page]({{site.baseurl}}/tutorials/assistantandskilldeploymentsteps.md) from the folder where your have cloned the GitHub repo.
+To deploy your services using the default configuration, from the folder where your have cloned the GitHub repo follow the steps either for [C#]({{site.baseurl}}/tutorials/csharp/create-skill/4_provision_your_azure_resources.md) or for [TypeScript]({{site.baseurl}}/tutorials/typescript/create-skill/4_provision_your_azure_resources/.md).
```
Comment (dfavretto) [inline docs/_docs/reference/skills/phone.md]: Would do mind preserving the content of the line, and just add the two link for both C# and TypeScript?
```suggestion
To deploy your services using the default configuration, follow the steps in this common deployment documentation page([C#]({{site.baseurl}}/tutorials/csharp/create-skill/4_provision_your_azure_resources.md)/[TypeScript]({{site.baseurl}}/tutorials/typescript/create-skill/4_provision_your_azure_resources/.md)) from the folder where you have cloned the GitHub repo.
```

File: docs/_docs/reference/skills/productivity-todo.md [hunk 17]
```diff
@@ -139,4 +139,4 @@ HomeworkSynonym | homework, home work |
     ]
 ```
 
-1. Redeploy your To Do Skill.
+3. Redeploy your To Do Skill.
```
Comment (dfavretto) [inline docs/_docs/reference/skills/productivity-todo.md]: Due to difference in the indentation, the numbered list is wrong, could you fix this too?

**Comment by dfavretto** [comment]
Also, could you check the documents that have some reference to Botskills? As some have some misleading information, i.e. in the tutorial on [adding a Skill to the VA](https://github.com/microsoft/botframework-solutions/blob/master/docs/_docs/tutorials/typescript/create-skill/6_add_your_skill.md) the command described is using the parameter `--cs` but this is a `TypeScript` tutorial, it should be using `--ts` instead.

File: docs/_docs/reference/skills/phone.md (hunk 16 above)
Comment (VictorGrycuk) [reply docs/_docs/reference/skills/phone.md]: That sounds weird to me, but ok

File: docs/_docs/reference/skills/productivity-todo.md (hunk 17 above)
Comment (VictorGrycuk) [reply docs/_docs/reference/skills/productivity-todo.md]: I will fix the numbering and add indent the content of the list to improve the readability of the list (the numbers get lost in the content)

File: docs/_docs/reference/skills/automotive.md [hunk 18]
```diff
@@ -169,33 +169,33 @@ Follow the instructions below to add the Automotive Skill to an existing Virtual
...
+      "luPath": "../skills/automotiveskill/automotiveskill/CognitiveModels/LUIS/en/settings_value.lu"
+    },
+    ```
+
+- Add dispatch references to the core LUIS intents for the skill within the **assistant/CognitiveModels/en/dispatch.lu** file as shown below. Only the vehicle settings model is required for dispatch. This enables the Dispatcher to understand your new capabilities and route utterances to your skill
+  
+	```
+	# l_Automotive
+	- [VEHICLE_SETTINGS_CHANGE](../../../../skills/automotiveskill/automotiveskill/CognitiveModels/LUIS/en/settings_dispatch.lu#VEHICLE_SETTINGS_CHANGE)
+	```
```
Comment (dfavretto) [inline docs/_docs/reference/skills/automotive.md]: Be careful, it seems with this change the indentation broke. Please take a look in the bullet points inside the step 1:
image

Comment (VictorGrycuk) [reply docs/_docs/reference/skills/automotive.md]: Yes, the indentation was all funky.
Now both the code fence and the bullet should be aligned accordingly.

**Pull request closed without merging**